spec/support/command_helpers.rb

```diff
@@ -296,6 +296,40 @@ def set_date
     send_command_and_confirm @task, command_list, "intention to set date"
   end
 
+  def verify_synchronization
+    @site.log "Set date", level: :test
+    command_list = build_command_list :M0104, :setDate, {
+      securityCode: SECRETS['security_codes'][1],
+      year: "2020",
+      month: "9",
+      day: "29",
+      hour: "17",
+      minute: "29",
+      second: "51"
+    }
+    send_command_and_confirm @task, command_list, "intention to set date"
+    request_status_and_confirm "current date and time",
+      { S0096: [
+        :year,
+        :month,
+        :day,
+        :hour,
+        :minute,
+        :second,
+      ] }
+    @site.log "Reset date", level: :test
```

Comment:
if resetting the clock is important to avoid other tests to fail, you should probably en ensure block to rest the clock even if some earlier part of the code fails

Reply:
Smart, I'll do that.